src/main/java/io/jenkins/plugins/checks/github/GitHubChecksPublisher.java

```diff
@@ -61,30 +65,48 @@ public void publish(final ChecksDetails details) {
             GitHubAppCredentials credentials = context.getCredentials();
             GitHub gitHub = Connector.connect(StringUtils.defaultIfBlank(credentials.getApiUri(), gitHubUrl),
                     credentials);
-
-            GitHubChecksDetails gitHubDetails = new GitHubChecksDetails(details);
-
-            Optional<Long> existingId = context.getId(gitHubDetails.getName());
-
-            final GHCheckRun run;
-
-            if (existingId.isPresent()) {
-                run = getUpdater(gitHub, gitHubDetails, existingId.get()).create();
+            boolean skipPublish = false;
+            String contributor = context.getContributor();
+            buildLogger.log("contributor name is " + contributor);
+            if (contributor.isEmpty()) {
+                skipPublish = false;
+            } else {
```

Comment:
```suggestion
            if (!contributor.isEmpty()) {
```
the `then` branch was redundant